Makefile

```diff
@@ -58,9 +58,10 @@ test-hyperkit: $(LINUXKIT) test-initrd.img test-bzImage test-cmdline
 	$(call check_test_log, test.log)
 
 .PHONY: test-gcp
+test-gcp: export CLOUDSDK_IMAGE_NAME?=test
```

Comment:
AFAICT it won't work any other way!
see: https://www.gnu.org/software/make/manual/make.html#Target_002dspecific for example usage...

If I try and put this on the target line, I get:
```
Makefile:62: *** commands commence before first target.  Stop.
```